test/integration/sign_in_test.rb

```diff
@@ -133,5 +154,38 @@ class SignInTest < ActionDispatch::IntegrationTest
       assert_response_contains "entered too many times"
       assert_equal 1, EventLog.where(event: EventLog::TWO_STEP_LOCKED, uid: @user.uid).count
     end
+
+    should "not permit an expired cookie to be used to bypass 2SV" do
+      visit root_path
+      signin_with_2sv(email: "email@example.com", password: "some passphrase with various $ymb0l$")
+      remember_2sv_session = Capybara.current_session.driver.request.cookies["remember_2sv_session"]
+
+      Timecop.travel(30.days.from_now + 1) do
+        visit root_path # Clears the expired cookie from the Capybara session
+        Capybara.current_session.driver.request.cookies["remember_2sv_session"] = remember_2sv_session
+
+        visit root_path
```

Comment:
It only cleared it as it is expired. The newly-set one won't be expired. `visit root_path` could be any path. Server isn't clearing the cookie, it's just forcing the Capybara driver to check which cookies are still valid.